### PR TITLE
Add subdomain config in cookie sharing topic

### LINKED ISSUE #7392: How those app share cookies ?
I'm confusing with this doc,  AFAK  cookies has domain ,  and there nowhere to configurate ,  and  if they live in differenct sub domain , how can the cookie be shared ?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 99307104-c0d7-3e82-f0b2-af61dcfe072a
* Version Independent ID: 315c02cb-54c4-e386-da85-6e4311ca10c2
* Content: [Share cookies among apps with ASP.NET and ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/cookie-sharing?view=aspnetcore-2.1&tabs=aspnetcore2x)
* Content Source: [aspnetcore/security/cookie-sharing.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/security/cookie-sharing.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #7392

### CODE REVIEW

**Comment by guardrex** [comment]
@scottaddie Since Chris is OOF this week, I think we can go by his remark on the issue as to the sufficiency of these updates. :point_right: https://github.com/aspnet/Docs/issues/7392#issuecomment-402808388

File: aspnetcore/security/cookie-sharing.md [hunk 1]
```diff
@@ -45,6 +45,12 @@ In the `ConfigureServices` method, use the [ConfigureApplicationCookie](/dotnet/
...
 
+When hosting apps that share cookies across subdomains, specify a common domain in the [Cookie.Domain](/dotnet/api/microsoft.aspnetcore.http.cookiebuilder.domain) property. To share cookies across apps at `contoso.com`, such as `first_subdomain.contoso.com` and `second_subdomain.contoso.com`, specify the `Cookie.Domain` as `.contoso.com`:
+
+```csharp
+options.Cookie.Domain = ".contoso.com";
+```
+
 See the *CookieAuthWithIdentity.Core* project in the [sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/security/cookie-sharing/sample/) ([how to download](xref:tutorials/index#how-to-download-a-sample)).
 
 # [ASP.NET Core 1.x](#tab/aspnetcore1x/)
```
[the remark this answers is not in the document]
Comment (scottaddie) [reply aspnetcore/security/cookie-sharing.md]: I'm okay with that approach.

[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/security/cookie-sharing.md]: https://github.com/aspnet/Docs/issues/7794

[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/security/cookie-sharing.md]: My plan is to make another big dent in 2.1 sample updates this sprint. I have the mega-Configuration issue to work on this sprint. Other than that tho, it looks like I'll have time for some good 2.1 sample update work.

**Pull request merged**